{
    "affected": [
        {
            "package": {
                "ecosystem": "Photon OS:3.0",
                "name": "python3-setuptools",
                "purl": "pkg:rpm/vmware/python3-setuptools?distro=photon-3"
            }
        },
        {
            "package": {
                "ecosystem": "Photon OS:3.0",
                "name": "python3-setuptools",
                "purl": "pkg:rpm/vmware/python3-setuptools?distro=photon-3"
            },
            "ranges": {
                "events": [
                    {
                        "introduced": "0"
                    },
                    {
                        "fixed": "57.4.0-2.ph3"
                    }
                ],
                "type": "ECOSYSTEM"
            }
        },
        {
            "package": {
                "ecosystem": "Photon OS:3.0",
                "name": "python3-setuptools",
                "purl": "pkg:rpm/vmware/python3-setuptools?distro=photon-3"
            },
            "ranges": [
                {
                    "events": [
                        {
                            "introduced": "0"
                        },
                        {
                            "fixed": "57.4.0-2.ph3"
                        }
                    ],
                    "type": "ECOSYSTEM"
                }
            ]
        }
    ],
    "id": "PHSA-2024-3.0-0786",
    "modified": "2025-01-24T05:27:06Z",
    "published": "2024-08-27T00:00:00Z",
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://github.com/vmware/photon/wiki/Security-Update-3.0-786"
        }
    ],
    "related": [
        "CVE-2024-6345"
    ]
}